yt_dlp/extractor/stripchat.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import (
+    compat_str,
+)
+from ..utils import (
+    ExtractorError,
+    lowercase_escape,
+    try_get,
+)
+
+
+class StripchatIE(InfoExtractor):
+    _VALID_URL = r'https?://stripchat\.com/(?P<id>[0-9A-Za-z-_]+)'
+    _TEST = {
+        'url': 'https://stripchat.com/feel_me',
+        'info_dict': {
+            'id': 'feel_me',
+            'ext': 'mp4',
+            'title': 're:^feel_me [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}$',
+            'description': 're:^.*$',
```

Comment:
using that regex makes no sense
```suggestion
            'description': str,
```